changelog/README.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 8th January 2025
+* Document webhook event `PaymentUpdated` in [General Webhooks](../events/wh-general.md).

```

Comment:
```suggestion
* Added webhook event `PaymentUpdated` in [General Webhooks](../events/wh-general.md).

```

Reply:
Do you want Added? Because it was already there for over a year or two and now we have added it to the documentation.
I've changed this based on this comment https://github.com/MewsSystems/gitbook-connector-api/pull/710#discussion_r1905370117

You want to keep word Added instead of Document?

Reply:
Ah, sorry @VBuben, I missed that earlier comment from @jnv. Hmm, tricky, you could argue it's still being "Added" to the official API as described in the documentation. Or we could say "Added missing..." to imply that it was there but missing from the docs. Or perhaps "Documented missing..." to mean the same thing. I suppose if we wanted to be 100% clear we should say "Added missing documentation for...". What do you think? I think "Documented" on its own isn't quite clear, because that applies to everything.

Reply:
`Added missing documentation for` seems really good to me! 🚀 
@jnv objections?

Reply:
Agreed, sounds good!